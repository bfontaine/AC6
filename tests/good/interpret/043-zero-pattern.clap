val f = case { 0 => 1 | _ => 2 }
val z = f (A[1])


