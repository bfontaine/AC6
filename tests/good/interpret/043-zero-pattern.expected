:- f = <code>
:- z = 2
